app/jobs/courses/clone_course_job.rb

```diff
@@ -0,0 +1,11 @@
+module Courses
+  class CloneCourseJob < ApplicationJob
+    queue_as :low_priority
+
+    def perform(source_course_id)
+      course = Course.find(source_course_id)
+      new_name = [course.name, "copy"].join(" - ")
```

Comment:
Could you please move this to `en.yml`. Perhaps `jobs.courses.clone_course.new_name`?

Reply:
sure